presto-docs/src/main/sphinx/connector/thrift.rst

```diff
@@ -42,6 +42,11 @@ Property Name                                 Description
 ``static-location.hosts``                     Location of Thrift servers
 ``presto-thrift.max-response-size``           Maximum size of a response from thrift server
 ``presto-thrift.metadata-refresh-threads``    Number of refresh threads for metadata cache
+``presto-thrift.max-retry-attempts``          Number of retries before finally fails a thrift request
```

Comment:
It seems like "..before finally fails.." is not the right working. Maybe, ".. before thrift request finally fails"? @electrum please suggest a better wording.

Reply:
Maybe "Maximum number of retry attempts for failed retryable thrift requests"? borrowed from AWS's document :-) :
http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/ClientConfiguration.html#DEFAULT_RETRY_POLICY